source/rpt/parameters_rpt.cc

```diff
@@ -190,6 +173,21 @@ Parameters::DetectorParameters::declare_parameters(ParameterHandler &prm)
                       "none",
                       Patterns::FileName(),
                       "Detector positions file name");
+
+    prm.declare_entry("dead time",
+                      "1",
+                      Patterns::Anything(),
```

Comment:
```suggestion
                                                Patterns::List(Patterns::Double()),
```
The right pattern is a list of doubles and not anything. So you can actually specify that what you want and need is a list of doubles :)
See this part in Lethe for more details : https://github.com/lethe-cfd/lethe/blob/cda6095c0c3f2df9c2e0fb9e1dfca967d350536d/source/core/parameters_lagrangian.cc#L318